css/dropkick.scss

```diff
@@ -101,7 +102,8 @@ $dk-disabled-color: #BBBBBB !default;
 }
 
 .dk-select-options {
-  width: 100%;
+  min-width: 100%;
```

Comment:
Properties should be ordered background-color, border, border-radius, list-style, margin, max-height, min-width, overflow-x, overflow-y, padding, width, z-index